task5/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
.idea, *.iml should be in a git ignore

use 
```
git rm -r .idea
git rm *.iml
```
to remove them from git index but leave in file system